### PR TITLE
chore: added style lint for directionality

### LINKED ISSUE #8979: Lint rules for directionality
With 3.0, we're updating the style layer to use only css rules that work in both directions. Devs can easily forget about this, which is why we like to introduce some additional lint rules on the CSS to avoid this.

See https://stylelint.io/user-guide/rules/combine#-allowed-list--disallowed-list-color-named-and-applicable--no--rules for more information.

### PR DESCRIPTION
**Pull request by LTiger14**
Closes #8979

### CODE REVIEW

File: feature-libs/organization/administration/styles/_form.scss [hunk 1]
```diff
@@ -39,7 +39,6 @@
     @include media-breakpoint-down(sm) {
       flex: 100%;
     }
-    padding-right: 20px;
```
Comment (tobi-or-not-tobi) [inline feature-libs/organization/administration/styles/_form.scss:42]: should you not bring this back with a logical style prop?

File: .stylelintrc.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (tobi-or-not-tobi) [inline .stylelintrc.json:1]: what about `text-align: left;`, can we disallow individual values as well?

Comment (tobi-or-not-tobi) [inline .stylelintrc.json:1]: Another file polluting the root of our file system. Can we move this to a folder? I can see us moving a bunch of files that aren't so important to everyone further down.

Comment (tobi-or-not-tobi) [reply .stylelintrc.json:1]: Can we add `float`?

File: .stylelintrc.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "rules": {
+    "property-disallowed-list": [
+      "margin-left",
+      "margin-right",
+      "padding-left",
+      "padding-right",
+      "border-left",
+      "border-right",
+      "border-inline",
```
Comment (tobi-or-not-tobi) [inline .stylelintrc.json:10]: not used often, but what's wrong with this one?

**Review by tobi-or-not-tobi — CHANGES_REQUESTED** [review]
few remarks

File: feature-libs/organization/administration/styles/_form.scss (hunk 1 above)
Comment (LTiger14) [reply feature-libs/organization/administration/styles/_form.scss:42]: Yes I should, I missed it.

File: .stylelintrc.json (hunk 2 above)
Comment (LTiger14) [reply .stylelintrc.json:1]: Screen Shot 2020-10-27 at 12 25 12 PM

😢

File: .stylelintrc.json (hunk 3 above)
Comment (LTiger14) [reply .stylelintrc.json:10]: The `border-inline` & `border-inline-color` & `border-inline-style` (same for block) don't have adequate level of browser support.

You should use: `boder-inline-end` & `border-inline-start` if it's only one side. If you want a border all around or on both sides you can use the `border` shorthand.

File: .stylelintrc.json (hunk 2 above)
Comment (LTiger14) [reply .stylelintrc.json:1]: I can move the file but for some reason the vscode extension can't pick up files that aren't in the root. So if we move it we can't use it.

Comment (tobi-or-not-tobi) [reply .stylelintrc.json:1]: I don't want us to use it actually. I see 2 notions of it in our css, we could drop those, or delay this request.

File: .stylelintrc.json (hunk 3 above)
Comment (tobi-or-not-tobi) [reply .stylelintrc.json:10]: gotcha

**Review by tobi-or-not-tobi — DISMISSED** [review]
LGTM

**Review by tobi-or-not-tobi — APPROVED** [review]

**Pull request merged**